### ISSUE TITLE
occ files_external:notify exits spontaneously

### ISSUE THREAD

**Issue by sm8ps** [nextcloud/server#10097]
### Steps to reproduce
1. Set up local folder as external storage. The folder only contains a handful of tiny test files for testing purposes.
2. Activate app INotify file watcher by @icewind1991 (and have requirements installed as per its README.md).
3. Run occ files_external:notify <id>

### Expected behaviour
Judging from its name, I would assume that files_external:notify should keep watching file changes indefinitely. This seems consistent with the remark in README.md: "Note that this command runs continuously and should be started in the background by an init system or other task manager for best usage."

### Actual behaviour
External file changes are detected, indeed. By the next run of cron.php, changes are then synched down to the clients. However, the script exits after roughly 60 seconds (sometimes almost to the second, sometimes some 30 seconds more). After that, changes are not synched to the client anymore.

### Server configuration

**Operating system**: Ubuntu 18.04 running in an LXC on a Ubuntu 16.04 host.

**Web server:** Nginx 1.15.0

**Database:** MariaDB (mysql  Ver 15.1 Distrib 10.3.7-MariaDB,)

**PHP version:** 7.2

**Nextcloud version:** (see Nextcloud admin page) 13.4.0

**Updated from an older Nextcloud/ownCloud or fresh install:** fresh install

**Where did you install Nextcloud from:** tar-ball

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - activity: 2.6.1
  - comments: 1.3.0
  - dav: 1.4.7
  - federatedfilesharing: 1.3.1
  - federation: 1.3.0
  - files: 1.8.0
  - files_external: 1.4.1
  - files_inotify: 0.1.2
  - files_pdfviewer: 1.2.1
  - files_sharing: 1.5.0
  - files_texteditor: 2.5.1
  - files_trashbin: 1.3.0
  - files_versions: 1.6.0
  - files_videoplayer: 1.2.0
  - firstrunwizard: 2.2.1
  - gallery: 18.0.0
  - groupfolders: 1.2.1
  - logreader: 2.0.0
  - lookup_server_connector: 1.1.0
  - nextcloud_announcements: 1.2.0
  - notifications: 2.1.2
  - oauth2: 1.1.1
  - password_policy: 1.3.0
  - provisioning_api: 1.3.0
  - serverinfo: 1.3.0
  - sharebymail: 1.3.0
  - survey_client: 1.1.0
  - systemtags: 1.3.0
  - theming: 1.4.5
  - twofactor_backupcodes: 1.2.3
  - updatenotification: 1.3.0
  - workflowengine: 1.3.0
Disabled:
  - admin_audit
  - encryption
  - user_external
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "lxcnextcloud.okunterhof5.ch",
            "lxcNextCloud",
            "nextcloud.wildi.name"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "http:\/\/lxcnextcloud.okunterhof5.ch",
        "dbtype": "mysql",
        "version": "13.0.4.0",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "logtimezone": "Europe\/Zurich",
        "loglevel": "2",
        "log_rotate_size": 10485760,
        "skeletondirectory": "",
        "share_folder": "\/Mit mir geteilt",
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "tls",
        "mail_smtpauth": 1,
        "mail_smtpport": "587",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "default_language": "de",
        "trashbin_retention_obligation": "disabled",
        "versions_retention_obligation": "disabled",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***"
    }
}

```
</details>

**Are you using external storage, if yes which one:** local (s. above)

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Mozilla Firefox Quantum 60.0.2 (64-Bit)

**Operating system:** Ubuntu 16.04

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
No errors reported at log level 3 during the testing period.
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Command is run from command line.
```
</details>

**Comment by sm8ps** [comment]
I had overlooked an entry in nextcloud.log where it is choking at some open_basedir issue (addressed in a comment to #8717). The behavior described above remains unchanged even with open_basedir completely disabled in /etc/php/7.2/cli/php.ini which makes nextcloud.log remain completely silent.

**Comment by cowai** [comment]
@sm8ps If you had the issue I had with the connection the DB dying due to timeout, then its fixed in latest minor releases of at least NC 15 and 16.
